metabox/metabox/scenarios/config/config_files/chains/etc_checkbox.conf

```diff
@@ -0,0 +1,10 @@
+[config]
+config_filename=includes.conf
+[manifest]
+# overwrite a set by some.conf
+a = 0
```

Comment:
0 doesn't say anything, I'd much more prefer "value from etc checkbox conf".
ofc this applies to all below.